.github/workflows/run-tx-tool.yaml

```diff
@@ -0,0 +1,97 @@
+name: Run tx-tool from ECR
+
+on:
+  workflow_dispatch:
+    inputs:
+      zcash_node_address:
+        description: "Zcash node address"
+        required: false
+        default: "dev.zebra.zsa-test.net"
+      zcash_node_port:
+        description: "Zcash node port"
+        required: false
+        default: "18232"
+      zcash_node_protocol:
+        description: "Zcash node protocol"
+        required: false
+        default: "http"
+      image_tag:
+        description: "Docker image tag"
+        required: false
+        default: "latest"
+  schedule:
+    - cron: "0 */2 * * *" # Runs every 2 hours
+
+env:
+  AWS_REGION: ${{ vars.AWS_REGION }}
+  ECR_REPOSITORY: ${{ vars.ECR_REPOSITORY }}
+
+jobs:
+  run-container:
+    name: Run Container from ECR
+    runs-on: ubuntu-latest
+    environment: dev
+
+    steps:
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: ${{ env.AWS_REGION }}
```

Comment:
Can we do this without relying on AWS secrets/our accounts?

Forcing this dependancy will prevent others to copy this action.